presentation/src/main/java/com/example/presentation/ui/components/Divider.kt

```diff
@@ -0,0 +1,28 @@
+package com.example.presentation.ui.components
+
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material.Divider
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+import com.example.presentation.ui.theme.DividerGray
+
+@Composable
+fun HorizontalDivider(
+    modifier: Modifier = Modifier,
+    color: Color = DividerGray,
+    thickness: Dp = 1.dp,
+    verticalPadding: Dp = 0.dp
+) {
+    Divider(
+        modifier = modifier
+            .fillMaxWidth()
+            .height(thickness)
+            .padding(vertical = verticalPadding),
+        color = color
+    )
```

Comment:
```suggestion
    modifier: Modifier = Modifier,
    color: Color = DividerGray,
    thickness: Dp = 1.dp,
    verticalPadding: Dp = 0.dp
) {
    Divider(
        modifier = modifier
            .fillMaxWidth()
            .padding(vertical = verticalPadding),
        color = color,
        thickness = thickness
    )
```

thickness 속성이 Divider에 있을텐데 따로 적용안하신 이유가 혹시 있을까요?? 
아마 Divider 내부에서 height로 넣어주긴 하지만요 ㅋㅋㅋ 저는 위처럼 Divider에게 그대로 역할을 위임해주는게 좋을 것 같아요